client/src/pages/mainPage/components/home/Home.css

```diff
@@ -51,3 +51,39 @@
 section {
 	margin-top: 8rem;
 }
+
+/* Adem */
+.message {
+	color: blue;
+}
+
+.message:hover {
+	background-color: yellow;
+}
+
+.gradient-custom-2 {
+	/* fallback for old browsers */
+	background: #fccb90;
+  
+	/* Chrome 10-25, Safari 5.1-6 */
+	background: -webkit-linear-gradient(to right, #ee7724, #d8363a, #dd3675, #b44593);
+  
+	/* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+	background: linear-gradient(to right, #ee7724, #d8363a, #dd3675, #b44593);
+  }
+  
+  @media (min-width: 768px) {
+	.gradient-form {
+	  height: 100vh !important;
```

Comment:
Try and avoid `important` usage, Adam - https://uxengineer.com/css-specificity-avoid-important-css/

Reply:
Thanks @Luke-Rogerson

Reply:
@ademalagoz Hase this issue been resolved?